src/main/java/com/zendesk/maxwell/MaxwellAbstractRowsEvent.java

```diff
@@ -193,7 +195,11 @@ public RowMap jsonMap() {
 				Column c = colIter.next();
 				ColumnDef d = defIter.next();
 
-				Object value = c.getValue();
+				if ( c instanceof DatetimeColumn )
+					value = ((DatetimeColumn) c).getLongValue();
```

Comment:
how does this work? If the value is of type "YYYY-MM-DD HH:MM:SS" ? How are we getting a long value of this?